### ISSUE TITLE
PostgreSQL instructions do not include archive_command/restore_command

### ISSUE THREAD

**Issue by OrangeDog** [wal-g/wal-g#1287]
### Database name
PostgreSQL

# Issue description

https://github.com/wal-g/wal-g/blob/v2.0.0/docs/PostgreSQL.md

No details are given on how to configure PostgreSQL. I assume, based on wal-e, that it should be this (if env variables are set):

```
archive_command = 'wal-g wal-push %p'
restore_command = 'wal-g wal-fetch %f %p'
```

**Comment by x4m** [comment]
Yes, you are right, but restore_command = '/usr/bin/wal-g wal-fetch "%f" "%p'
and you can use something like wal-g wal-fetch "%f" "%p" --config /etc/wal-g/wal-g.yaml

Docs PR would be very welcome!
